### DISCUSSION TITLE
Missing ww entities with RC35/BK13 on 3.4.0b7

### DISCUSSION THREAD

**Discussion by g6094199** [emsesp/EMS-ESP32#1002] [General]
every entity binary_sensor.boiler_ww* is missing on HA and the EMS GUI

which is everything regarding warm water beside boiler_ww_3_way_valve_on_ww

on the latest 3.3.x stable WW regarding entities are there.

![Bildschirmfoto vom 2022-02-25 11-03-05](https://user-images.githubusercontent.com/37833326/155698738-60483d77-33b6-45ae-9fb7-aa4be8fa5ae7.png)

Also the BC10/RFM20 which is shown in the 3.3 GUI is gone in 3.4

console still  shows the contoller

> 
> show devices
> These EMS devices are currently active:
> 
> Boiler: BK13/BK15/Smartline/GB1x2 (DeviceID:0x08, ProductID:64, Version:02.07)
>  Received telegram type IDs: 0x10 0x11 0x15 0x1C 0x18 0x19 0x1A 0x35 0x34 
>  Fetched telegram type IDs: 0x14 0x16 0x33 
>  Pending telegram type IDs: 0xC2 0x26 0x2A 0xD1 0xE3 0xE4 0xE5 0xE6 0xE9 0xEA 
> 
> Thermostat: RC35 (DeviceID:0x10, ProductID:86, Version:01.16)
>  Received telegram type IDs: 0xA3 0x06 0xA2 0x12 0x42 0x48 0x47 0x52 0x51 0x5C 0x5B 0x38 0x39 
>  Fetched telegram type IDs: 0x3E 0x3D 0x3F 0xA5 0x37 
>  Pending telegram type IDs: 0x49 0x4C 0x53 0x56 0x5D 0x60 
> 
> Controller: BC10/RFM20 (DeviceID:0x09, ProductID:68, Version:02.00)

**Comment by MichaelDvP** [comment]
When upgrading from v3.3 to v3.4 do the following steps:
1.  reload page in browser or close and reopen browser to update all scripts.
2.  navigate to mqtt settings and disable mqtt-discovery, Click `SAVE`, 
3.  Wait 5 minutes (MINUTES not secondes),
4. Enable mqtt-discovery and click `SAVE`

The controller is not shown on webpage, it never has any value. 
ww-entities are renamed to `dhw`, the above procedure ensures that the ww's are removed and dhw's are created.

**Comment by proddy** [comment]
this is good, I think we need to add these steps to the releasenotes and doc so everyone is clear of the breaking changes and steps to go from 3.3 and 3.4

and we should release 3.4 soon
